packages/angular/src/generators/application/lib/add-e2e.ts

```diff
@@ -16,6 +24,32 @@ export async function addE2e(tree: Tree, options: NormalizedSchema) {
       standaloneConfig: options.standaloneConfig,
       skipPackageJson: options.skipPackageJson,
       rootProject: options.rootProject,
+      devServerTarget: 'serve-static',
```

Comment:
I'm not sure if we should change the targets to always use the static file server or only in the CI configuration. 

Personally, I think we should only update the CI configuration for the e2e tests to use the static server as locally using the static server to iterate on work provides a poor experience, since you have to wait for a full build.